sass/patternfly-react/_autocomplete.scss

```diff
@@ -0,0 +1,202 @@
+@import 'mixins';
+
+.rbt {
+  outline: none;
+
+  // Apply Bootstrap focus styles
+  & .focus {
+    @include box-shadow(inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(102, 175, 233, 0.6));
+
+    border-color: #66afe9;
```

Comment:
Color settings should use patternfly variables. See https://github.com/patternfly/patternfly/blob/master/src/sass/converted/patternfly/_color-variables.scss